test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -18,6 +18,17 @@
 // CHECK:   mutating func callAsFunction(_ x: Int32, _ y: Int32) -> Int32
 // CHECK: }
 
+// CHECK: struct HasPostIncrementOperator {
+// CHECK: }
+
+// CHECK: struct HasPreIncrementOperatorWithAnotherReturnType {
+// CHECK:   func successor() -> HasPreIncrementOperatorWithAnotherReturnType
+// CHECK: }
+
+// CHECK: struct HasPreIncrementOperatorWithVoidReturnType {
+// CHECK:   func successor() -> HasPreIncrementOperatorWithVoidReturnType
```

Comment:
Shouldn't this function be mutating and just update the value in `lhs.value` or something? Feels odd that a void function in Cxx is being mapped into a function that returns a type (a copy I assume)

Reply:
We don't want to map them differently for a couple reasons:
* We're importing `operator++` mainly to make it possible to increment C++ iterators from Swift. If the signature of `func successor() -> Self` changes depending on the return type of `operator++`, we can't conform iterators to a single protocol (`UnsafeCxxInputIterator`), and that defeats the purpose of importing `operator++` in the first place.
* In most C++ use cases, there is no difference between `void operator++()` and `T& operator++()`. I think it would be a bit confusing to introduce it in Swift.